docs/js-core-3/week-2/lesson.md

```diff
@@ -425,37 +425,95 @@ fetch(_____)
+Tom: “Hi Katie, would you mind spending a few minutes looking at this function I am working on? I can’t seem to find what’s wrong with the code.”
 
-:::note Exercise (30 minutes)
+Katie: “I am extremely busy and stressed with a deadline, but let me have a look.”
+
+Tom: “Here is the function. Do you see anything wrong with it?”
+
+Katie: After looking at the code for some time… “Seriously? How could you miss this? You are missing an End statement. You need to be more careful when writing code so you only ask for help for things that are not this obvious.”
+
+Katie walks away.
+
+*Situation 2*: Patricia stops by Carla’s desk.
+
+Patricia: “Hi Carla. Just wanted to let you know how much I appreciate the work you did in the last project and I hope we get to work together again”
+
+Carla: “Oh Thank You! You made my day!”
+
+*Situation 3*: Team members in a project retrospective meeting where they discuss things they did well and things that need improvement.
+
+Jessica: “I think overall the project went well but developers were constantly waiting for Karen to finish testing their code before they could deploy to Production. She was overwhelmed with the amount of work on her plate”
+
+Karen: “Yes, I felt bad for blocking deployments but I cannot give my approval until I have completed testing the code.”
+
+Jessica: “Did you ask for help? We can have developers take the role of QA tester when you are overwhelmed.”
+
+Karen: “No, I did not ask for help. I did not think of this as an option. I will start implementing this strategy going forward”
+
+Volunteer to share your role-playing experience with the class.
+
+:::
 
-**Exercise objective:** To practice estimating effort for user stories
+:::note Exercise (20 min)
 
-Work in small groups. For this sizing example, you have 5 stories. The first one is already t-shirt sized as a Medium. To size story #2, the team must compare it against story #1 - just as you’d compare t-shirts to find their size - Is it bigger or smaller?
+**Exercise objective:** To practice turning destructive feedback into constructive feedback.
 
-As a team, you ask yourselves “is story #2 bigger or smaller than #1?” If you decide it takes less effort, then you decide how much smaller it is. Enough to be a S, or much smaller to be a XS?
+Work in small groups. Turn the feedback below into constructive feedback. Try to be creative.
 
-You do the same for each of the following stories. "Is story #3 bigger or smaller than story #1?” If it’s bigger, by how much? Is it double the size? (XL) or just slightly bigger? (L).
+*Feedback 1*: After a weak sales pitch, Ryan’s manager blamed Ryan for what went wrong. The manager told Ryan that he had no idea what he was doing. Ryan’s personality was being questioned rather than his work. The feedback provided no helpful insights to help Ryan perform better.
```

Comment:
To me the "weak sales pitch" doesn't really give enough detail on the issue to be able to turn this into constructive feedback. I worry it would turn into "Ryan hasn't done anything wrong", which I don't think is quite what we're going for here.

```suggestion
*Feedback 1*: After an important deadline was missed, Ryan’s manager blamed Ryan for not working fast enough. The manager told Ryan that he had no idea what he was doing. Ryan’s personality was being questioned rather than his work. The feedback provided no helpful insights to help Ryan perform better.
```